src/clr-angular/popover/dropdown/_menu-mixins.clarity.scss

```diff
@@ -43,7 +43,7 @@
   text-transform: none;
 
   &:hover,
-  &:focus {
+  &.focus {
```

Comment:
Do you know if we are supposed to `.clr-` prefix the `focus` class? I started out without then added it in combobox.

Reply:
I think you're right, I'll update that to `.clr-focus`.